lectures/arellano.md

```diff
@@ -0,0 +1,923 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.5
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Default Risk and Income Fluctuations
+
++++
+
+In addition to what's in Anaconda, this lecture will need the following libraries:
+
+```{code-cell} ipython3
+!pip install quantecon
+```
+
+## Overview
+
+This lecture computes versions of  Arellano’s  [[Are08](https://python-advanced.quantecon.org/zreferences.html#id126)]
```

Comment:
@chappiewuzefan , this is not the right way to do citations.  Please read https://manual.quantecon.org/styleguide/references.html